docs/development.md

```diff
@@ -145,10 +152,11 @@ components in various configurations and is useful for example to develop and do
 
 See a running guide as example [https://rems-dev.rahtiapp.fi/guide](https://rems-dev.rahtiapp.fi/guide).
 
-## Automated accessibility test report
 
-We use [axe](https://www.deque.com/axe/) for automated accessibility tests.
+## Mac OS installation for Apple M1 Chip (Apple Silicon M1)
+
+    1. Java Installation: install Java via Azul, make sure you download the version of [Java for MacOS for ARM architecture and not newer then version 13](https://www.azul.com/downloads/zulu-community/?version=java-13-mts&os=macos&architecture=arm-64-bit&package=jdk). 
+ 
+    2. Do not install leiningen through homebrew, rather use the script from [here](https://purelyfunctional.tv/guide/how-to-install-clojure/#mac-leiningen) to install leiningen. Run `lein update 2.8.3` and `lein version` and make sure the version is  2.8.3.
```

Comment:
lein 2.9.1 and 2.9.5 also work for me. I don't think we depend on any specific lein version. Did you have problems with fresh leiningen?